sda-commons-server-opentelemetry/docs/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,163 @@
+# Migration Guide
+
+## Starter Bundle
+ If you do not use sda-commons-starter with [SdaPlatformBundle](../../sda-commons-starter/src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java), you need to remove the Jaeger bundle and OpenTracing bundle and add the OpenTelemetry bundle.
+
+Before:
+```java
+bootstrap.addBundle(JaegerBundle.builder().build());
+bootstrap.addBundle(OpenTracingBundle.builder().build());
+```
+
+After:
+```java
+bootstrap.addBundle(
+        OpenTelemetryBundle.builder()
+            .withAutoConfiguredTelemetryInstance()
+            .withExcludedUrlsPattern(Pattern.compile(String.join("|", excludedTracingUrls)))
+            .build());
+```
+
+## Replace OpenTracing dependencies to OpenTelemetry dependencies
+
+Before:
+```gradle
+testImplementation 'io.opentracing:opentracing-mock'
+```
+
+After:
+```gradle
+testImplementation 'io.opentelemetry:opentelemetry-sdk-testing'
+```
+
+## Executors need to be instrumented to follow traces when parts of the request are handled asynchronously.
+
+Before: 
+```java
+return new AsyncEmailSendManager(
+  new InMemoryAsyncTaskRepository<>(limits.getMaxAttempts()),
+  environment
+      .lifecycle()
+      .executorService("email-sender-%d")
+      .minThreads(1)
+      .maxThreads(limits.getMaxParallel())
+      .build(),
+```
```

Comment:
How did instrumentation work before? It almost looks like a new feature.

Reply:
Must have been magic. At least the traces where correlated.